resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -62,6 +64,11 @@ do_start()
     <%= action %>
     <% end -%>
 
+    if [ ! -d "/var/run/puppetlabs/${REALNAME}" ] ; then
```

Comment:
FWIW, in the ezbake.conf file in Puppet Server, we had ![pre-start-actions](https://github.com/puppetlabs/puppetserver/blob/puppet-server-2.6.0/resources/ext/ezbake.conf#L54) which did this:

```
/usr/bin/install --directory --owner={{user}} --group={{user}} --mode=775 /var/run/puppetlabs/puppetserve
```
The one advantage of doing it this way is that the permissions would be corrected even if the directory already existed at the time this code were run.

I'm open to doing it this way instead, though, if you think that's better.